### ISSUE TITLE
Error in phi function

### ISSUE THREAD

**Issue by martialblog** [fau-klue/pandas-association-measures#13]
The new phi function throws an error when used in the tests. 

```
def test_log_likelihood(sample_dataframe):
    
        df = sample_dataframe
        am.log_likelihood(sample_dataframe)

 ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

I created a test to recreate the error here: https://github.com/fau-klue/pandas-association-measures/tree/phi-error

**Comment by martialblog** [comment]
@ausgerechnet Hi, can you have a look at that please?

**Comment by ausgerechnet** [comment]
all AMs now take a dict-like object (a dataframe row) as argument, *not* a dataframe. correct usage is thus `df.apply(am.log_likelihood, axis=1)`

**Comment by ausgerechnet** [comment]
fixed in v1.0.5, you can use `am.log_likelihood(sample_dataframe)` again
